app/assets/stylesheets/components/_navbar.scss

```diff
@@ -0,0 +1,55 @@
+.navbar-toggle {
+  display: none;
+
+  // @include media($screen-tablet) {
+  // }
+}
+
+.navbar-brand {
+  font-size: 2rem;
```

Comment:
Properties should be ordered background, color, display, font-size, line-height, margin, padding-left